drivers/usb/device/usb_dc_kinetis.c

```diff
@@ -474,15 +474,15 @@ int usb_dc_ep_is_stalled(const u8_t ep, u8_t *const stalled)
 		*stalled = dev_data.ep_ctrl[ep_idx].status.in_stalled;
 	}
 
-	if (SYS_LOG_LEVEL == SYS_LOG_LEVEL_INFO) {
-		u8_t bd_idx = get_bdt_idx(ep,
-				dev_data.ep_ctrl[ep_idx].status.in_odd);
-		bd_idx = bd_idx;
-		SYS_LOG_WRN("active bd ctrl: %x", bdt[bd_idx].set.bd_ctrl);
-		bd_idx = get_bdt_idx(ep,
-				~dev_data.ep_ctrl[ep_idx].status.in_odd);
-		SYS_LOG_WRN("next bd ctrl: %x", bdt[bd_idx].set.bd_ctrl);
-	}
+#if defined(CONFIG_SYS_LOG) && (SYS_LOG_LEVEL >= SYS_LOG_LEVEL_WARNING)
```

Comment:
I think this change is due to the logic issue in include/logging/sys_log.h, meybe we can fix it there directly

Reply:
What is the logic issue in `include/logging/sys_log.h`?

Reply:
the SYS_LOG_WRN is defined in sys_log.h, but it wil lbe none defined in below condition
define CONIG_SYS_LOG  and SYS_LOG_LEVEL < SYS_LOG_LEVEL_WARNING.
so in the definition of SYS_LOG_WRN, it shall not be 
{code}
#if (SYS_LOG_LEVEL >= SYS_LOG_LEVEL_WARNING)
#define SYS_LOG_WRN(...) LOG_COLOR(SYS_LOG_TAG_WRN,                     \
        SYS_LOG_COLOR_YELLOW, ##__VA_ARGS__)
#endif
{code}
but
{code}
#if (SYS_LOG_LEVEL >= SYS_LOG_LEVEL_WARNING)
#define SYS_LOG_WRN(...) LOG_COLOR(SYS_LOG_TAG_WRN,                     \
        SYS_LOG_COLOR_YELLOW, ##__VA_ARGS__)
#else  
 #define SYS_LOG_WRN(...) { ; }
#endif

{code}